packages/contracts/test/core/V3/GenArt721CoreV3_ProjectSplitters.test.ts

```diff
@@ -0,0 +1,616 @@
+import { constants } from "@openzeppelin/test-helpers";
+import { expect } from "chai";
+import { ethers } from "hardhat";
+
+import {
+  T_Config,
+  getAccounts,
+  assignDefaultConstants,
+  deployAndGet,
+  deployCoreWithMinterFilter,
+  deployWithStorageLibraryAndGet,
+} from "../../util/common";
+
+import { Mock0xSplitsV2Splitter } from "../../../scripts/contracts";
+
+// test the following V3 core contract derivatives:
+const coreContractsToTest = [
+  "GenArt721CoreV3_Engine", // V3 core Engine contract
+  "GenArt721CoreV3_Engine_Flex", // V3 core Engine Flex contract
+];
+
+/**
+ * Tests for V3 core dealing with configuring splitters on projects.
+ */
+for (const coreContractName of coreContractsToTest) {
+  describe(`${coreContractName} Project Configure`, async function () {
+    async function _beforeEach() {
```

Comment:
Def not a blocker, but do we want to type the config a little more tightly here?

Reply:
good call - no more typing issues in my vscode for this file after the changes in [0d1c455](https://github.com/ArtBlocks/artblocks-contracts/pull/1455/commits/0d1c4556570bea61b6d8bdf6f30d138ae3931e1d) 🥳